client/my-sites/upgrades/domain-management/dns/dns-add-new.jsx

```diff
@@ -95,7 +98,7 @@ const DnsAddNew = React.createClass( {
 				if ( error ) {
 					notices.error( error.message );
```

Comment:
Ah, I see, thanks for the clarification. I need to finally read up properly on Redux...
Anyway, I'll test this first thing tomorrow :)